truffle/test/Owned.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Owned = artifacts.require('Owned');
+const truffleAssert = require('truffle-assertions');
+
+contract('Owned', (accounts) => {
+    let instance;
+
+    // Initiating new contract before each test
+    beforeEach("Creating new contract", async () => {
+        instance = await Owned.new({from: accounts[0]});
+    });
+
+    // Testing getOwner function.
+    it('Owner should be ' + accounts[0], () => {
```

Comment:
On the other hand, for `beforeEach` and `it`, I use the full `"should do", function() {` so that I have access to `this.timeout()` and `this.slow()` in the function's body.

Reply:
Thanks to point this out. However, what's the purpose to doing it if I don't need these functions?

Reply:
No point. Only, perhaps, that when you need them, you may need some time to rediscover why `this.slow()` "is not a function" when you use `() => {`.